### PR TITLE
Attachments in create ticket

### PR DESCRIPTION
**Pull request by jSylvestre**
Issue #223

### CODE REVIEW

File: Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx [hunk 1]
```diff
@@ -111,6 +112,21 @@ export const TicketContainer = () => {
             </div>
           </div>
         </div>
+          <FormGroup>
+              <Label>Want to attach any files?</Label>
+              <FileUpload
+                  files={ticket.attachments || []}
+                  setFiles={(f) => setTicket({ ...ticket, attachments: [...f] })}
```
Comment (srkirkland) [inline Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx:119]: Need to change this to using the lambda method like you did on the next line. Always use the lambda if you are going to use the state value in your setter.

**Review by srkirkland — CHANGES_REQUESTED** [review]

File: Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx (hunk 1 above)
Comment (jSylvestre) [reply Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx:119]: Ok, but I was just copying you code :)

File: Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx [hunk 2]
```diff
@@ -116,7 +116,7 @@ export const TicketContainer = () => {
               <Label>Want to attach any files?</Label>
               <FileUpload
                   files={ticket.attachments || []}
-                  setFiles={(f) => setTicket({ ...ticket, attachments: [...f] })}
+                  setFiles={(f) => setTicket((tick) => ({ ...tick, attachments: [...f] }))}
                   updateFile={(f) =>
```
Comment (jSylvestre) [inline Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx:120]: @srkirkland Like this?

Comment (srkirkland) [reply Harvest.Web/ClientApp/src/Tickets/TicketContainer.tsx:120]: Yup looks good

**Review by srkirkland — APPROVED** [review]

**Pull request merged**